packages/hooks/src/__tests__/useResidenceSelfDeclaration.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useResidenceSelfDeclaration from '../useResidenceSelfDeclaration';
+
+describe('useResidenceSelfDeclaration', () => {
+    test('should return true if client residence is spain', async () => {
```

Comment:
@shaheer-deriv can we update the description for this test case?
Lets not make anything specific to spain. 
Something like:
test(' should return true if client's residence require self-declaration', async ()

Reply:
surest thing in 🇦🇪 today